map/bookmark.cpp

```diff
@@ -182,418 +156,396 @@ size_t BookmarkCategory::FindBookmark(Bookmark const * bookmark) const
 
 namespace
 {
+string const PLACEMARK = "Placemark";
```

Comment:
As you touched this code, would you be so kind to rename this constants in accordance with our style guide, like PLACEMARK -> kPlacemark, STYLE -> kStyle, STYLE_MAP -> kStyleMap, etc. ?